services/main_menu_service.rb

```diff
@@ -0,0 +1,56 @@
+class MainMenuService
+  extend InputHelper
+  extend OutputHelper
+  extend Uploader
+
```

Comment:
Oh my ... , ALL methods in this class get `accounts` and `current_account` as arguments, don't you think it would be better to create an instance of this service and not pass it every time ?

Reply:
Yes, it's much better. Done